feature-libs/product/test.ts

```diff
@@ -6,8 +6,8 @@ import {
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting,
 } from '@angular/platform-browser-dynamic/testing';
-import 'zone.js/dist/zone';
-import 'zone.js/dist/zone-testing';
+import 'feature-libs/organization/node_modules/zone.js/dist/zone';
+import 'feature-libs/organization/node_modules/zone.js/dist/zone-testing';
```

Comment:
Why the different lib now uses zone.js from organization library?

I've seen similar changes in:
```
integration-libs/cdc/src/test.ts
projects/storefrontapp/src/polyfills.ts
.... (and many more)
```